cmd/vpc/vmnic/port/public.go

```diff
@@ -6,26 +6,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const _CmdName = "port"
+const cmdName = "port"
 
 var Cmd = &command.Command{
-	Name: _CmdName,
+	Name: cmdName,
 
 	Cobra: &cobra.Command{
-		Use:     _CmdName,
+		Use:     cmdName,
 		Aliases: []string{"sw"},
 		Short:   "VPC switch management",
 	},
 
 	Setup: func(self *command.Command) error {
 		subCommands := command.Commands{
-		//create.Cmd,
-		//destroy.Cmd,
-		//list.Cmd,
+			//create.Cmd,
```

Comment:
File is not gofmt-ed with -s

Reply:
`goimports` and `gofmt` fight each other on this. The CI tool needs to stop...